src/transport/SecureSession.cpp

```diff
@@ -20,6 +20,11 @@
 namespace chip {
 namespace Transport {
 
+OperationalId SecureSession::GetPeerOperationalId() const
+{
+    return OperationalId(mPeerNodeId, GetFabricIndex());
```

Comment:
This is going to crash as soon as someone calls GetPeerOperationalId() on a PASE session before AddNOC, no?  Like in ReadHandler?

Which is probably why the cirque tests are all failing, and why non-on-network commissioning would fail, etc.